src/flatpak-spawn.c

```diff
@@ -263,7 +263,7 @@ main (int    argc,
 
   context = g_option_context_new ("COMMAND [ARGUMENT…]");
 
-  g_option_context_set_summary (context, "Flatpak portal spawn");
+  g_option_context_set_summary (context, "Run a command in a sandbox");
```

Comment:
Well, sometimes not in sandbox too =)

Reply:
it would be just as right to say: Run a command outside the sandbox

Reply:
Yeah, the --host argument kinda totally changes this, but i think it makes sense to document what it does by default.